apps/web/src/views/Swap/V3Swap/components/Paymaster/config/config.ts

```diff
@@ -0,0 +1,30 @@
+import { zkSync } from 'viem/zksync'
+import { NATIVE_CURRENCY_ADDRESS } from 'views/Swap/MMLinkPools/constants'
+import { zksyncTokens } from '@pancakeswap/tokens'
+import { PaymasterToken } from '../types'
+
+export const DEFAULT_PAYMASTER_TOKEN: PaymasterToken = {
+  address: NATIVE_CURRENCY_ADDRESS,
+  decimals: 18,
+  name: 'Ether',
+  symbol: 'ETH',
+  logoURI: 'https://assets.pancakeswap.finance/web/native/324.png',
+  isNative: true,
+  isToken: false,
+  chainId: zkSync.id,
+}
+
+export const paymasterTokens: PaymasterToken[] = [
+  { ...zksyncTokens.wbtc, logoURI: 'https://tokens.pancakeswap.finance/images/symbol/wbtc.png' },
```

Comment:
We can reuse the `CurrencyLogo` component from internal widgets so I don't think we need this config anymore?

Reply:
sorry I mean the config of the `logoURI`